libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -189,7 +189,6 @@ bool AC_AutoTune::init_internals(bool _use_poshold,
 // stop - should be called when the ch7/ch8 switch is switched OFF
 void AC_AutoTune::stop()
 {
-    // set gains to their original values
```

Comment:
no big deal but might as well restore the comment as well..

Reply:
Fixed